LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs

```diff
@@ -3818,6 +3818,54 @@ private static void GetNuvotonConfigurationD(ISuperIO superIO, Manufacturer manu
 
                         break;
 
+                    case Model.ROG_STRIX_Z790_E_GAMING_WIFI: //NCT6798D
+
+                        // Volts
+                        v.Add(new Voltage("CPU Core Voltage", 0));
+                        v.Add(new Voltage("+5V", 1));
+                        v.Add(new Voltage("AV (Standby)", 2));
+                        v.Add(new Voltage("+3.3V", 3));
+                        v.Add(new Voltage("+12V", 4));
+                        v.Add(new Voltage("AUXTIN4 (VIN8)", 5));
+                        v.Add(new Voltage("AUXTIN0 (VIN4)", 6));
+                        v.Add(new Voltage("+3.3V (Standby)", 7));
+                        v.Add(new Voltage("CMOS Battery", 8));
+                        v.Add(new Voltage("VTT", 9));
+                        v.Add(new Voltage("Memory Controller", 10));
+                        v.Add(new Voltage("CPU L2 Cache", 11));
+                        v.Add(new Voltage("PCH 1.05V", 12));
+                        v.Add(new Voltage("CPU System Agent", 13));
+                        v.Add(new Voltage("CPU Input Voltage", 14));
```

Comment:
Nit pick here - `CPU Input Auxiliary` to be a tiny bit more specific

Reply:
No problem. I will change that.